source/_patterns/00-base/_transitions.scss

```diff
@@ -6,5 +6,6 @@
 //$transition-fade
 //$btn-transition
 @import '~bootstrap/scss/forms';
```

Comment:
@rloos289, this is a copy/paste error and will result in duplicate form output. Kill it.